examples/custom_contentviews.py

```diff
@@ -0,0 +1,60 @@
+import string
+from libmproxy import script, flow, utils
+import libmproxy.contentviews as cv
+from netlib.http import Headers
+import lxml.html
+import lxml.etree
+
+
+def start(context, argv):
+    class ViewPigLatin(cv.View):
```

Comment:
Yes - just move the class to the top level please. It doesn't have to live in the start hook. :smiley: